lib/scarlet/helpers/nick_helper.rb

```diff
@@ -0,0 +1,57 @@
+class Scarlet
+  module NickHelper
+    def handle_special_nick(name)
+      case name
+      when ':me'
```

Comment:
I think it's safe to unprefix these, assuming we will not have a `me`/`you` user. I should be able to do `remind me <...>`, `poke me`, and so on.